apps/condo/domains/ticket/utils/serverSchema/ticketBlankExport.js

```diff
@@ -0,0 +1,593 @@
+                            text: i18n.ticketWithNumberAndDated,
+                        },
+                        {
+                            text: `${i18n.page} ${currentPage}`,
+                            alignment: 'right',
+                        },
+                    ],
+                },
+            ]
+        },
+        content: [
+            // ticket number & organization name
+            {
+                marginBottom: 12,
+                columns: [
+                    {
+                        text: `${i18n.ticket} №${blank.number}`,
+                        color: '#222222',
+                        fontSize: 15,
+                        bold: true,
+                    },
+                    {
+                        text: blank.organization,
+                        font: 'BoldOpenSans',
+                        fontSize: 10,
+                        alignment: 'right',
+                        marginTop: 5,
+                        color: '#707695',
+                    },
+                ],
+            },
+
+            // address
+            {
+                marginBottom: 12,
+                stack: [
+                    {
+                        style: 'subtitle',
+                        text: i18n.address,
+                    },
+                    {
+                        text: blank.address,
+                        font: 'BoldOpenSans',
+                        fontSize: 9,
+                    },
+                ],
+            },
+
+            // client
+            {
+                marginBottom: 12,
+                stack: [
+                    {
+                        style: 'subtitle',
+                        text: i18n.clientName,
+                    },
+                    {
+                        columnGap: 20,
+                        columns: [
+                            {
+                                text: blank.clientName,
+                                font: 'BoldOpenSans',
+                                fontSize: 9,
+                                width: '*',
+                            },
+                            {
+                                text: blank.clientPhone,
+                                font: 'BoldOpenSans',
+                                fontSize: 9,
+                                alignment: 'right',
+                                width: 120,
+                            },
+                        ],
+                    },
+                ],
+            },
+
+            // details
+            {
+                marginBottom: 12,
+                stack: [
+                    {
+                        style: 'subtitle',
+                        text: i18n.details,
+                    },
+                    {
+                        text: blank.details,
+                        fontSize: 8,
+                    },
+                ],
+            },
+
+            // comments
+            addComments(blank.comments),
```

Comment:
I think, `render…` prefix is more clear here, because we are transforming representation from one to another.

```suggestion
            renderComments(blank.comments),
```